src/components/MultiSelect/MultiSelect.stories.tsx

```diff
@@ -10,6 +10,7 @@ export default {
     component: MultiSelectComponent,
     args: {
         type: MultiSelectType.Default,
+        size: 'm',
```

Comment:
I think it would be better for readability to actually use the full word 'small', 'medium', etc... instead of just one letter. (even if it's totally understandable 😁 )